Client/Clique/Common/Constants/Constants.swift

```diff
@@ -6,13 +6,43 @@
 //
 
 import Foundation
+import Alamofire
+import UIKit
 
 enum Constants {
     
     enum Service {
         private static let baseURL = "https://cliquewebservice20221115180920.azurewebsites.net/api/"
         static let registerURL = baseURL.appending("Authentication/RegisterUser")
-        static let loginURL = baseURL.appending("Authentication/LoginUser")    
+        static let loginURL = baseURL.appending("Authentication/LoginUser")
+        static let profileGetUserURL = baseURL.appending("User")
+        static let profileUpdateURL = baseURL.appending("User/UpdateUserData")
+        static let passwordUpdateURL = baseURL.appending("User/UpdateUserPassword")
+        
+        static func getToken() -> String {
+            guard let token = UserStorage.token else { return "" }
+            return token
```

Comment:
This may create a potential bug because `""` is not a valid token.
You can return a `String?` instead of `String`, or you can even remove this method.

I'll make a suggestion for `authorizationHeader` method ;)